src/bionics.cpp

```diff
@@ -147,10 +147,9 @@ bool bionic_data::is_included( const bionic_id &id ) const
     return std::find( included_bionics.begin(), included_bionics.end(), id ) != included_bionics.end();
 }
 
-bionic_data::bionic_data()
+bionic_data::bionic_data() : name( translation( "bad bionic" ) ),
```

Comment:
```suggestion
bionic_data::bionic_data() : name( "bad bionic" ),
```
Same for the other member.

Reply:
It is necessary for extracting the strings for translation.

Reply:
Do those strings really need to be translatable? They should never appear, and if they appear it indicates a bug and should stand out.

Anyway, if you keep it as it is now, you should add a comment. Otherwise someone might remove it as it appears to be redundant.

Reply:
Alright, I made it `no_translation` so it will properly ignore the string during translation and not appear redundant.